src/components/notification/channel/googlechat.ts

```diff
@@ -0,0 +1,233 @@
+/**********************************************************************************
+ * MIT License                                                                    *
+ *                                                                                *
+ * Copyright (c) 2021 Hyperjump Technology                                        *
+ *                                                                                *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy   *
+ * of this software and associated documentation files (the "Software"), to deal  *
+ * in the Software without restriction, including without limitation the rights   *
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      *
+ * copies of the Software, and to permit persons to whom the Software is          *
+ * furnished to do so, subject to the following conditions:                       *
+ *                                                                                *
+ * The above copyright notice and this permission notice shall be included in all *
+ * copies or substantial portions of the Software.                                *
+ *                                                                                *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     *
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       *
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE    *
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER         *
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,  *
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE  *
+ * SOFTWARE.                                                                      *
+ **********************************************************************************/
+import axios from 'axios'
+
+import { NotificationMessage } from '../../../interfaces/notification'
+import { GoogleChatData } from '../../../interfaces/data'
+import { log } from '../../../utils/pino'
+
+export const sendGoogleChat = async (
+  data: GoogleChatData,
+  message: NotificationMessage
+) => {
+  const notifType =
+    message.meta.type[0].toUpperCase() + message.meta.type.substring(1)
+  let chatMessage
+
+  switch (message.meta.type) {
+    case 'start':
```

Comment:
I think for the "start" case, the `subtitle` should be "Monika is starting" just like the Teams version.
![image](https://user-images.githubusercontent.com/16670475/140696874-60a93e5a-e1fc-49c0-8b5f-4116262e2cf7.png)


And for the sections, it only prints the message body. Time is not needed as it is already at the top:
![image](https://user-images.githubusercontent.com/16670475/140696648-2408014b-581d-4816-9e7e-545493e42530.png)